src/components/ui/VanCard.jsx

```diff
@@ -3,16 +3,26 @@ import { Link } from "react-router-dom";
 const VanCard = ({ imageUrl, name, price, type, id }) => {
   return (
     <Link to={`/vans/${id}`} className="hover:opacity-80">
```

Comment:
Please if it possible you can use relative path. like this  
```suggestion
    <Link to={id} relative="path" className="hover:opacity-80">
```